packetbeat/sniffer/sniffer.go

```diff
@@ -436,7 +437,9 @@ func (s *sniffer) sniffHandle(ctx context.Context, handle snifferHandle, dec *de
 			}
 		}
 
-		logp.Debug("sniffer", "Packet number: %d", packets)
+		if s.config.OneAtATime {
```

Comment:
Can you add a note for this change in the commit message or here?

Reply:
I thought it would be quite wasteful of resources to have this executing for every received packet. It seems like this is way to noisy to be useful. But I assume that it was useful to give feedback if you were stepping through packets using `-O` like

```
Press enter to read packet

{"log.level":"debug","@timestamp":"2023-05-12T23:01:36.248Z","log.logger":"sniffer","log.origin":{"file.name":"sniffer/sniffer.go","file.line":441},"message":"Packet received.","service.name":"packetbeat","network.packets":62,"ecs.version":"1.6.0"}
```